package.json

```diff
@@ -260,3 +260,4 @@
     "windosu": "^0.3.0"
   }
 }
+
```

Comment:
This PR simply adds a blank line at the end of the `package.json` file. The `npm-shrinkwrap.json` file is unchanged and the dependencies are the same as in v11.33.3. I have tested that this is enough to change the hash code of the `balena.exe` file (part of the standalone zip package) so that the AVG antivirus on Windows no longer identifies `balena.exe` as containing malware. I believe this is evidence that the AVG report was a false positive.